backend/entities/ContactMethod.cs

```diff
@@ -75,16 +75,15 @@ public class ContactMethod : BaseAppEntity
         /// <param name="organization"></param>
         /// <param name="methodType"></param>
         /// <param name="value"></param>
-        public ContactMethod(Person person, Organization organization, ContactMethodType methodType, string value)
+        public ContactMethod(Person person, Organization organization, string methodTypeId, string value)
```

Comment:
Create an override in another constructor to support this.

Reply:
Could do, important to realize that this way causes errors during inserts.

Reply:
Most likely errors related to not passing a ContactMethodType from the active Context.

Reply:
right, I was trying to create the contact method during mapping which wasn't possible using the above.

Reply:
Regardless, I'll create the overload.